CameraXBasic/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -36,6 +37,7 @@
     <fragment
         android:id="@+id/camera_fragment"
         android:name="com.android.example.cameraxbasic.fragments.CameraFragment"
+        tools:layout="@layout/camera_ui_container"
```

Comment:
The fixes the navigation graph preview in Android Studio.

Reply:
This seems like a bad idea, since the layout is loaded dynamically. If the layout is changed, this will be out of date and that will be very confusing. I suggest reverting this change.

Reply:
Now it looks alike this again:
![screenshot_01](https://user-images.githubusercontent.com/736481/72762459-edb48d00-3bdf-11ea-9174-950d6803dd56.png)

Reply:
I understand the idea, I'm just not sure about the value it adds. We can discuss in a separate PR, thanks for the clarifying screenshots!